src/lib/pickles/impls.ml

```diff
@@ -131,18 +131,9 @@ module Step = struct
   let input ~proofs_verified ~wrap_rounds =
     let open Types.Step.Statement in
     let spec = spec proofs_verified wrap_rounds in
-    let (T (typ, f)) =
-      Spec.packed_typ
-        (module Impl)
-        (T
-           ( Shifted_value.Type2.typ Other_field.typ_unchecked
```

Comment:
Does the change from typ_unchecked to typ have any impacts, or no because we call this closure no matter what?

Reply:
I printed out the circuit and it seemed unchanged. I suspect it's an artifact of past iteration